mitodl/open-discussions
# Added a setting to change the default for feature flags

#### What are the relevant tickets?
N/A

#### What's this PR do?
Adds a setting (default to `False` as before) for setting the default for all feature flags.

#### How should this be manually tested?
Comment out a feature flag setting in `.env` and verify it is still enabled (default set in docker-compose.yml)
